### PR TITLE
Portal beam don't detonate tank anymore

### LINKED ISSUE #8209: portal gun oversight
**Problem Description**:
What is the problem?
you can make fuel tanks go boomy if you shoot one with the _bluespace wormhole projector_no its a portal gun

**What did you expect to happen**: 
Why do you think this is a bug?
cause its not a laser,its not a bullet, its a hole(worm)

**What happened instead**:
How is what happened different from what you expected?
it got bommy

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?
since people with portal guns arent ubvious that they will make harm but people with shotguns and e guns are

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
make a portal gun
find your victim fueltank
shoot it with your portal gun
get bwoinked by an admin because you made the fuel tank go bommmy

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.

### PR DESCRIPTION
**Pull request by Anasari**
Fixes #8209 

Add in a no_damage tag for portal beam, change damage from 3 to 0. Not like that 3 damage matter at all.

🆑Ansari
Portal beam no longer detonate fuel tank.
/🆑

### CODE REVIEW

File: code/modules/projectiles/projectile/special.dm [hunk 1]
```diff
@@ -203,9 +203,10 @@
 	name = "bluespace beam"
 	icon_state = "spark"
 	hitsound = "sparks"
-	damage = 3
+	damage = 0
 	var/obj/item/weapon/gun/energy/wormhole_projector/gun
 	color = "#33CCFF"
+	nodamage = 1
```
Comment (Birdtalon) [inline code/modules/projectiles/projectile/special.dm]: Bool? `FALSE`

**Comment by Anasari** [comment]
And done!

**Comment by Anasari** [comment]
That was an accidentally

**Comment by FalseIncarnate** [comment]
You accidentally the whole thing?

**Comment by Anasari** [comment]
I accidentally pressed close and comment instead of comment

**Pull request merged**